be/src/exec/arrow/arrow_reader.h

```diff
@@ -100,6 +100,7 @@ class ArrowReaderWrap {
     int get_cloumn_index(std::string column_name);
 
     void prefetch_batch();
+    bool is_case_sensitive() {return _caseSensitive; }
```

Comment:
How about to use `_case_sensitive` instead?

Reply:
I'll fix it.